### ISSUE TITLE
Add the Ribbon XML files under source control

### ISSUE THREAD

**Issue by eteichm** [rubberduck-vba/Rubberduck#3755]
I would like to be able to add the Ribbon XML files under source control, in the same repo as the vba source code.  [enhancement] 
Related to #3369

**Comment by rubberduck203** [comment]
I'm assuming these aren't exported along with the source code when using the "Export All" feature?
We've remove git integration (#3758), but the export feature is still there.

Would you like us to reword this request to "Export all exports Ribbon XML files"?

**Comment by eteichm** [comment]
yes, as git integration has been removed, let's reword it to "Export all Ribbon XML files". I'll do it if I can.
